src/scripts/cronjobs/firstDataBrokerRemovalFixed.tsx

```diff
@@ -0,0 +1,124 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import { OnerepScanResultRow, SubscriberRow } from "knex/types/tables";
+import {
+  getPotentialSubscribersWaitingForFirstDataBrokerRemovalFixedEmail,
+  markFirstDataBrokerRemovalFixedEmailAsJustSent,
+} from "../../db/tables/subscribers";
+import { initEmail, sendEmail } from "../../utils/email";
+import { renderEmail } from "../../emails/renderEmail";
+import { FirstDataBrokerRemovalFixed } from "../../emails/templates/firstDataBrokerRemovalFixed/FirstDataBrokerRemovalFixed";
+import { getEmailL10n } from "../../app/functions/l10n/cronjobs";
+import { sanitizeSubscriberRow } from "../../app/functions/server/sanitize";
+import { refreshStoredScanResults } from "../../app/functions/server/refreshStoredScanResults";
+import { getLatestOnerepScanResults } from "../../db/tables/onerep_scans";
+
+type SubscriberFirstRemovedScanResult = {
+  subscriber: SubscriberRow;
+  firstRemovedScanResult: OnerepScanResultRow;
+};
+
+function isFulfilledResult(
+  result: PromiseSettledResult<SubscriberFirstRemovedScanResult | undefined>,
+): result is PromiseFulfilledResult<SubscriberFirstRemovedScanResult> {
+  return typeof result !== "undefined" && result.status === "fulfilled";
+}
+
+void run();
+
+async function run() {
```

Comment:
I think the duplication here is OK for this PR since it's only the second script we've done like this, but we should look for opportunities to share code with `src/scripts/cronjobs/monthlyActivity.tsx`. I am imagining that as soon as we fix a bug or change behavior in one it's going to be easy to forget about the other.

Reply:
That’s a good point and I was considering this. I’ve refrained from touching anything related to the `monthlyActivity` mail as we are still observing its behavior and I did not want to interfere in any way. As soon as we confirm that we have our cronjobs run stable I’m happy to unify them more.